cmd/ocm/gcp/create-wif-config.go

```diff
@@ -123,40 +93,58 @@ func createWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 		return nil
 	}
 
-	gcpClientWifConfigShim := NewGcpClientWifConfigShim(GcpClientWifConfigShimSpec{
-		GcpClient: gcpClient,
-		WifConfig: wifConfig,
-	})
-
-	if err := gcpClientWifConfigShim.GrantSupportAccess(ctx, log); err != nil {
```

Comment:
Removing this line would be enough to resolve the issue until it can be properly addressed